packages/flutter_driver/lib/src/driver/timeline_summary.dart

```diff
@@ -155,13 +158,37 @@ class TimelineSummary {
       final TimelineEvent beginEvent = events.current;
       if (events.moveNext()) {
         final TimelineEvent endEvent = events.current;
-        result.add(Duration(microseconds: endEvent.timestampMicros - beginEvent.timestampMicros));
+        result.add(extractor(beginEvent, endEvent));
       }
     }
 
     return result;
   }
 
+  /// Extracts Duration list that are reported as a pair of begin/end events.
+  ///
+  /// See: https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU
+  List<Duration> _extractBeginEndEvents(String name) {
+    return _extractDurations(
+      name,
+      (TimelineEvent beginEvent, TimelineEvent endEvent) =>
+          Duration(microseconds: endEvent.timestampMicros - beginEvent.timestampMicros),
```

Comment:
nit - => should be on one line, or changed to `{ }`.

Reply:
Done.